### ISSUE TITLE
Support of MSX-modem machines

### ISSUE THREAD

**Issue by zeilemaker54** [openMSX/openMSX#1453]
Are you interested in supporting MSX-modem machine like the Sony HB-T600, HB-T7 and the Mitsubishi ML-TS2 ?
I am in the process of disassembly of the firmware of these machine, from which I can deduct some of the hardware details.
It would be nice to add some minimal support, to get the MSX-modem BASIC and the TELCOM application working.

Regards,
Arjen Zeilemaker

**Comment by MBilderbeek** [comment]
Yes I am, at least to get the software working. Better than nothing :) Hardware details are always welcome.

**Comment by zeilemaker54** [comment]
HB-T600 hardware registers:
D.7F00	EQU	7F00H			; NCU register data
D$7F40	EQU	7F40H			; NCU register select
D.7FC0	EQU	7FC0H			; segment select / line status

ML-TS2 hardware registers:
D.7F00	EQU	7F00H			; NCU register 0
D.7F01	EQU	7F01H			; NCU register 1
D.7F02	EQU	7F02H			; NCU register 2
D.7F03	EQU	7F03H			; NCU register 3
D.7FC0	EQU	7FC0H			; segment select / line status

register definitions:
; 7FC0, read
;   b3-b0 = line status
; 7FC0, write
;   b2-b0 = ROM segment (4000H-5FFFH)

; NCU register 0, read
;   b0 =
;   b1 =
;   b2 =
;   b3 = break detected
;   b4 = long loop detected
;   b6 =
;   b7 = dial tone detected
; NCU register 1, read
;   b3 = empty transmitter
;   b2 = data in receiver
;   b1 =
;   b0 =
; NCU register 2, read
;   not used
; NCU register 3, read
;   receiver data

; NCU register 0, write
; NCU register 1, write
;   b7 = send break disabled
;   b3 = disable receiver
; NCU register 2, write
; NCU register 3, write
;   transmitter data
